aws_piper/aws_piper.py

```diff
@@ -1,22 +1,43 @@
 from __future__ import absolute_import
 from __future__ import print_function
+
 import json
 import os
 from datetime import datetime
 
 import yaml
 from awscli.customizations.datapipeline import translator
+from super_mario.s3_client import S3Client
 
 from .data_pipeline import DataPipelineClient
 
 
+def send_query_to_S3(cfg):
+    query_file = cfg.get('sql_query_file')
+    s3 = S3Client()
```

Comment:
This file only defines the data pipeline commands and workflows. Things like s3 clients should be defined elsewhere as it's the case in the `data_pipeline.py` file.